pykob/internet.py

```diff
@@ -91,9 +91,8 @@ def read(self):
                 pass  # ignore Ack packet
             elif nBytes == 496:  # code or ID packet
                 self.tLastListener = time.time()
-                # stnID, seqNo, *code = codePacketFormat.unpack(buf)  # works in Python 3.x
```

Comment:
Did you mean to use this style instead here, then, now that it's strictly Python 3-only?

Reply:
Good eye, @pwdirks, nothing gets past you!

Actually, I did try the Python 3-only approach, which I used to think was cleaner, but it failed with a list/tuple conflict. I could have dealt with that in turn, but it got a bit messy.

As long as it's working I'll leave it for someone with a better eye for style to take care of.

Thanks for the review!

Reply:
Fair enough.  Might want to add a question mark, then, or "... is supposed to work as well in Python 3?", or just leave as-is.  Your call.